src/components/Tables/Table.tsx

```diff
@@ -153,12 +164,8 @@ const Table = <Data,>({
 
     const [data, setData] = useState(inputData);
 
-    const [selectedCheckboxes, setSelectedCheckboxes] = useState(
-        new Array<boolean>(data.length).fill(false)
-    );
-
     const updateCheckboxes = (newSelection: boolean[]): void => {
-        setSelectedCheckboxes(newSelection);
+        setSelectedCheckboxes!(newSelection);
```

Comment:
Having a non-null assertion is generally an anti-pattern unless it's absolutely necessary.

Reply:
Have removed as part of checkbox refactoring